### DISCUSSION TITLE
Need your help!

### DISCUSSION THREAD

**Discussion by ibrahimali123** [MahmoudAshraf97/AutoencoderCompression#3] [General]
Hello Mahmoud,
I hope you are doing well.

@MahmoudAshraf97 

I need to ask you a small question about this code, I will be happy if you send me your email and I will send you.
ibrahim.ali.0403@gmail.com

Thanks a lot.

**Comment by MahmoudAshraf97** [comment]
Hello Ibrahim,
my personal email is hassouna97.ma@gmail.com.
Although I'd prefer the discussion to be public in case anyone has better answers or suggestions

**Reply by ibrahimali123** [reply]
Thanks a lot @MahmoudAshraf97 
I just send you an email now.
Thanks for your reply
